wiki/Bazel-Setup-Instructions-for-Linux.md

```diff
@@ -0,0 +1,67 @@
+## Instructions
+
+**The steps to install Bazel on Linux are:**
+1. Install Bazel 
+2. Install the OpenJDK 8
+3. Install Python 2 and make sure it is active in your environment
+4. Set the Android SDK
+5. Set up the environment to be able to build Oppia Android
```

Comment:
The steps here should match those below. So:

5. Prepare the build environment.
6. Verify the Android build.

Reply:
Done.